src/users/userSelectors.js

```diff
@@ -54,6 +61,9 @@ export const getUsersSansMe: Selector<User[]> = createSelector(
   (users, ownEmail) => users.filter(user => user.email !== ownEmail),
 );
 
+export const getOwnUser: Selector<User> = (state: GlobalState) =>
```

Comment:
Perhaps `getSelfUser`? You'd raised yesterday the idea of making the naming of own/self/me/etc. consistent, and I think "self" seems like the best single term for that.